dicom-archive/DICOM/DCMSUM.pm

```diff
@@ -1,19 +1,21 @@
-# ------------------------------ MNI Header ----------------------------------
-#@NAME       : DICOM::DCMSUM
-#@DESCRIPTION: deals with dicom summaries for archiving and other purposes
-#@EXPORT     : none
-#@EXPORT_OK  : none
-#@EXPORT_TAGS: none
-#@USES       : DICOM::DICOM
-#@REQUIRES   : 
-#@VERSION    : $Id: DCMSUM.pm 9 2007-12-18 22:26:00Z jharlap $
-#@CREATED    : 2006/03/18, J-Sebastian Muehlboeck
-#@MODIFIED   : sebas
-#@COPYRIGHT  : Copyright (c) 2006 by J-Sebastian Muehlboeck, McConnell Brain Imaging
-#              Centre, Montreal Neurological Institute, McGill University.
-#-----------------------------------------------------------------------------
-
 package DICOM::DCMSUM;
+
+=pod
+
+=head1 NAME
+
+DICOM::DCMSUM -- deals with DICOM summaries for archiving and other purposes
```

Comment:
maybe we can say Acrhives DICOM summaries (instead of deals with archive summaries?)